internal/compress/lz4_test.go

```diff
@@ -91,31 +53,36 @@ func TestNewLZ4(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           opts: nil,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           opts: nil,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "returns (Compressor, nil) when option is empty",
+			args: args{
+				opts: nil,
+			},
+			want: want{
+				want: &lz4Compressor{
+					gobc: func() (gob Compressor) {
+						gob, _ = NewGob()
+						return
+					}(),
+					compressionLevel: 0,
+					lz4:              lz4.New(),
+				},
+			},
+		},
+		{
+			name: "returns (nil, error) when option apply fails",
```

Comment:
I think we need `returns (Compressor, nil) when option apply success` test case

Reply:
the first test pattern covers your suggestion.
it has `defaultLZ4Opts` and apply it.

Reply:
But I think it misses the test case to set the custom option 🤔

Reply:
well...
OK, I'll try it.